dist/icon/icon.css

```diff
@@ -2819,6 +2819,9 @@ svg.icon--disabled {
   color: var(--color-foreground-disabled);
   fill: currentColor;
 }
+svg.icon--disabled[class*="-colored"] {
+  filter: var(--color-icon-disabled-filter);
```

Comment:
Does this not work for the default case as well?

Reply:
Yep, will do.